ui/hooks/useGasFeeEstimates.js

```diff
@@ -27,12 +29,18 @@ import { useSafeGasEstimatePolling } from './useSafeGasEstimatePolling';
  *
  * @returns {GasEstimates} GasEstimates object
  */
-export function useGasFeeEstimates() {
+export function useGasFeeEstimates(networkClientId) {
   const gasEstimateType = useSelector(getGasEstimateType);
   const gasFeeEstimates = useSelector(getGasFeeEstimates, isEqual);
```

Comment:
should this be using `networkClientId` or `chainId` or something in the selector